### PR TITLE
Bill Situ <Bill.Situ@Oracle.com

### PR DESCRIPTION
**Pull request by bsitu**
On branch cna/Oracle/CVE-2020-14874
 Changes to be committed:
	modified:   2020/14xxx/CVE-2020-14874.json

### CODE REVIEW

File: 2020/14xxx/CVE-2020-14874.json [hunk 1]
```diff
@@ -3,15 +3,65 @@
     "data_format": "MITRE",
```
Comment (cve-team) [inline 2020/14xxx/CVE-2020-14874.json:3]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/14xxx/CVE-2020-14874.json:3]: ("Expecting ',' delimiter: line 64 column 17 (char 3273)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by bsitu** [comment]
file updated.

**Pull request merged**